ruby/lib/ThreatExchange.rb

```diff
@@ -17,27 +17,14 @@ def initialize(access_token=nil)
       @baseurl = 'https://graph.facebook.com'
     end
 
-    def malware_analyses(filter={})
-      filter[:access_token] = @access_token
-      begin
-        response = RestClient.get "#{@baseurl}/malware_analyses",
-        { params: filter  }
-        result = JSON.parse(response)
-      rescue => e
-        puts e.inspect
-      end
+    def malware_analyses(params={})
```

Comment:
Not sure about the styleguide used here, but most Ruby styleguides prescribe spaces around `=` for the method arguments.